src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.panel ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+li {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  margin-right: 20px;
+}
+
+.panel a {
+  text-transform: uppercase;
+  text-align: center;
+  color: #000;
```

Comment:
Consider using the CSS variable `var(--blue-color)` for the color property here instead of the hardcoded value `#000`. This will ensure consistency and make it easier to update the color scheme in the future.